nuget/uap10.0/MainPage.xaml.txt.pp

```diff
@@ -1,13 +0,0 @@
-﻿<forms:WindowsPage
```

Comment:
Getting rid of this closes https://github.com/nunit/nunit.xamarin/issues/79

Reply:
Yes, this is another template update too.